projects/ngrx.io/content/guide/store/architecture.md

```diff
@@ -1 +1,418 @@
 # Architecture
+
+NGRX is based on the architectural pattern Redux. Redux in turn is based upon the general pattern Pub-Sub or Publish, Subscribe. This means we send a message to a central data store and a number of listeners on that data store are being told there is a new message. 
```

Comment:
As you read my reviews on grammar, you will find that I am big on brevity. These are just suggestions, but the end result should read a little better.

- Consider changing `are being told there is` to `are notified that there is`